bazel/external_crates.bzl

```diff
@@ -1,5 +1,5 @@
 """
-This module declares all direct rust dependencies.
+This module declares all direct rust dependencies!
```

Comment:
Was this change intended?

Reply:
No, I'll put it out as soon as current workflow run is done.